source/screens/App/components/TopNavBar.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Link } from 'react-router-dom'
+import { MenuItems } from '../shared/const'
+import './TopNavbar.css'
+
+const TopNavbar = ({ Router, userName, iconUrl }) => {
+  const renderNavBar = (
+    <div className="TopNavBarContainer">
+      <input type="search" id="nav_search" placeholder="Search" />
```

Comment:
What would we be searching at this point? I would probably not put a search bar here until we've fleshed out exactly what would be searched on. Having a universal search bar at the beginning of an app seems like it's inviting trouble because you have to support searching something. It's easier to add a feature than to remove one... Again, if that's what you want I'm cool with it. 👍

Reply:
True - actually not sure. I was following the comp, but I was wondering the same thing and wanted to talk to you and Yining about that